### PR TITLE
[RFR] updating configuration reference

### LINKED ISSUE #13: [Doc] The configuration reference seems not up to date
Tree builder for `cache`node have been recently changed, and the doc was not updated.
The old one :
```YAML
cache:
        enabled:              false
        type:                 doctrine
        service:              ~
```
The new one :
```YAML
cache:
        enabled:              false
        service:              ~
        adapter: 
            type:                 doctrine
```

### PR DESCRIPTION
**Pull request by salahm**
closes #13

### CODE REVIEW

File: src/Resources/doc/configuration_reference.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ csa_guzzle:
 
         # The maximum size of the body which should be stored in the profiler (in bytes)
         max_body_size:        65536 # Example: 65536
-
-    # Whether or not to enable the logger
-    logger:               false # Example: %kernel.debug%
+    logger:
+        enabled:              false
+        service:              null
```
Comment (csarrazi) [inline src/Resources/doc/configuration_reference.md]: You should replace `null` by `~`

**Comment by csarrazi** [comment]
No need to open an issue in addition to a PR. A PR is automatically considered as an issue on github :)

File: src/Resources/doc/configuration_reference.md (hunk 1 above)
Comment (salahm) [inline src/Resources/doc/configuration_reference.md]: Then it should be edited here too ? : https://github.com/csarrazi/CsaGuzzleBundle/blob/master/src/DependencyInjection/Configuration.php#L52

Comment (csarrazi) [inline src/Resources/doc/configuration_reference.md]: Nope. Not needed. That's just to be consistent with the rest of the documentation, which uses `~` to represent `null`.

Comment (salahm) [inline src/Resources/doc/configuration_reference.md]: Okay, it's updated.

**Comment by csarrazi** [comment]
Thanks for your help, @salahm! :)

**Pull request merged**